generic/generic_cmd-1.2.3.recipe

```diff
@@ -61,8 +61,8 @@ BUILD()
 
 INSTALL()
 {
-	mkdir -p $commandBinDir
-	cp -a objects/projectx $commandBinDir
+	install -d $commandBinDir
+	install -t objects/projectx $commandBinDir
```

Comment:
Oops, with `-t` it's the other way:
```diff
-	install -t objects/projectx $commandBinDir
+	install -t $commandBinDir objects/projectx
```
Alternatively, you can use `install` without the `-t`:
```diff
-	install -t objects/projectx $commandBinDir
+-	install objects/projectx $commandBinDir
```